src/ScalarPotentials/ElectricPotential.jl

```diff
@@ -40,4 +40,22 @@ function ElectricPotential(nt::NamedTuple)
     N = get_number_of_dimensions(grid)
     ElectricPotential{T, N, S, typeof(grid.axes)}( ustrip.(uconvert.(u"V", nt.values)), grid)
 end
-Base.convert(T::Type{ElectricPotential}, x::NamedTuple) = T(x)
\ No newline at end of file
+Base.convert(T::Type{ElectricPotential}, x::NamedTuple) = T(x)
+
+"""
+    get_ticks_at_positions_of_large_gradient(epot::ElectricPotential)
+
+The electric potential is analyzed in order to find and return ticks where the gradient (electric field) is strong.
+The electric field strength is the strongest at the position of the pn-junction at least for the 1D linear case.
+Thus, this function is likely to return ticks which are located close to the pn-junction of a semiconductor.
+"""
+function get_ticks_at_positions_of_large_gradient(epot::ElectricPotential{T, 3}) where T

```

Comment:
It is a pain to compare functions on GitHub preview: what is the difference between this function and `get_ticks_at_positions_of_edge_of_undepleted_volumes`. There seem to be quite some duplications (?) Can these functions we somehow merged where similar?

Reply:
The impurity scale field does not need to be normalized and the selection criterion is different.